spec.md

```diff
@@ -1076,6 +1076,36 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Enumerations
+
+Enumerations MUST follow the same guidelines as classes, except where otherwise noted here.
+
+Methods in Enumerations MUST follow the same guidelines as methods in classes.  Non-public methods MUST use `private`
+instead of `protected`.
+
+In the case of a backed enum, there MUST NOT be a space between the enum name and colon, and there MUST be exactly one
+space between the colon and the backing type.
+
+Enumeration cases MUST use CamelCase capitalization.
+
+Constants in Enumerations MAY use either CamelCase or UPPER_CASE capitalization.  CamelCase is RECOMMENDED,
+so that it is consistent with cases.
+
+No more than one case is permitted per line.
```

Comment:
The term "case" appears in this text with three different meanings:
1. "In the case of" referring to a hypothetical situation
2. "CamelCase ... consistent with cases." referring to type casing
3. "No more than one case" referring to `case` declarations.

Let's avoid use #1 by using terminology like "When using a backed enum," and switch to "case declaration" for use #3 to make it more plainly obvious what exactly is meant.